gencert-env.public.example.sh

```diff
@@ -0,0 +1,9 @@
+COMMON_NAME=Comune di Roma
+DAYS=3650
+ENTITY_ID=https://spid.comune.roma.it
+KEY_LEN=3072
+LOCALITY_NAME=Roma
+MD_ALG=sha512
+ORGANIZATION_IDENTIFIER=PA:IT-c_h501
+ORGANIZATION_NAME=Comune di Roma
+SPID_SECTOR=public
```

Comment:
Why not naming it `public.env.example`?

Reply:
for editor highlighting: having it with the `.sh` extension automatically use the correct one, with `.example` you have to set it manually every time. Should I change it?

Reply:
Local editor behavior/settings should not influence the source code. I would prefer to name it as suggested.